pkg/operator/cluster/cluster.go

```diff
@@ -16,33 +16,49 @@ limitations under the License.
 Some of the code below came from https://github.com/coreos/etcd-operator
 which also has the apache 2.0 license.
 */
-package operator
+package cluster
 
 import (
 	"fmt"
+	"time"
 
 	"k8s.io/client-go/pkg/api/errors"
 
+	"github.com/coreos/pkg/capnslog"
 	"github.com/rook/rook/pkg/cephmgr/client"
 	cephmon "github.com/rook/rook/pkg/cephmgr/mon"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/operator/api"
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	"github.com/rook/rook/pkg/operator/mon"
 	"github.com/rook/rook/pkg/operator/osd"
+	"github.com/rook/rook/pkg/operator/rgw"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/api/v1"
 )
 
+const (
+	defaultPool = "rook"
+)
+
+var (
+	healthCheckInterval = 10 * time.Second
+	logger              = capnslog.NewPackageLogger("github.com/rook/rook-operator", "op-cluster")
```

Comment:
i figured if there is a source file with the same name as the package it feels like this could go in there. i knew i'd be getting a question from you though!